recipe-server/client/control_new/components/data/QueryExtensions.js

```diff
@@ -3,27 +3,38 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 
-import { fetchAllExtensions } from 'control_new/state/extensions/actions';
+import * as extensionActions from 'control_new/state/extensions/actions';
```

Comment:
I notice this method is used in a few spots - maybe it requires a larger team discussion.